### ISSUE TITLE
HfLA: Design Meeting Agenda

### ISSUE THREAD

**Issue by kristine-eudey** [hackforla/website#2106]
### Dependency
When we get a new design lead

### Overview
This issue tracks the agenda for the UX team's weekly Thursday meetings.

### Agenda Template:
Meeting: (DATE)

Participants:

**Things to discuss:**

**Things discussed:**

**Actions to take:**

### Weekly Agenda
[Meeting 8/12/21](https://github.com/hackforla/website/issues/2106#issuecomment-898030721)
[Meeting 8/19/21](https://github.com/hackforla/website/issues/2106#issuecomment-900508307)
[Meeting 8/26/21](https://github.com/hackforla/website/issues/2106#issuecomment-906811158)

**Comment by kristine-eudey** [comment]
Meeting: 8/12/21

**Participants:**
Anh, Basu, Simone, Sophia, Ava, Saasha, Kristin, Christina, Kristine

**Things to discuss:**
- Weekly Standup: Are you currently working on an issue? 
      - If yes, what have you done on it since last Thursday?
       - Any blockers? Want feedback?
       - What do you plan to accomplish by next Thursday? 
       - If no please take an issue tonight. 
- Issues to assign
- Meetings - encouraging cameras to increase sense of connection, if you feel comfortable doing so 
- Research updates
- Homepage redesign updates: [current version](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=13809%3A92582) 

**Things discussed:**
- Basu: Gave update on donor/partner research plan
- Christina - Would like some input from the team on her Member Feature research. Will reach out via Slack 
- Sophia: Recently finished design issue for footer. Taking issue #2021 
- Saasha: Very close to being finished with mega-footer (issue #1969), will be finished for review at Sunday meeting. 
- Kristin - Waiting on feedback at Sunday meeting regarding issue #1979. Will work on continued homepage design with Kristine.
- Anh: Completed her recent issue. Taking issue #1051 
- Simone: Working on research plan. 
- Kristine - Gave update on current homepage redesign plan
- Sophia let us know that there's already been audits of all buttons on the site, from the dev perspective in issue #1921, and possibly from a design perspective in an issue Danielle made. 


**Actions to take:**
- [ ] Everyone to tap their networks for potential donor/partner research participants. Send email addresses of folks we could contact to see if they'd be willing to participate in a 30-45 minute interview to Basu. 
- [ ] Kristine will provide feedback to Saasha on footer in preparation for presentation at Sunday meeting. 
- [ ] Anyone interested in a homepage design work session on either Friday 8/13 or Saturday 8/14 please reach out to Kristine.

**Comment by kristine-eudey** [comment]
Meeting: 8/19/21

Participants: Kristine, Anh, Basu, Simone, Ava, Alyssa

**Things to discuss:**

- Updated process for completing an issue 
  - What to do when you're finished with an issue:
    - Make a comment on the issue alerting that you've completed the design portion of the issue.
    - Leave issue open and drag it to the UAT column. That is where design lead will look to approve completed issues to pass on to product & dev teams
  - What to do when you need guidance / have a question. 
  - When to use Questions / In-Review Column **vs**. when to add "Help wanted" label
- Any questions on GitHub process? Anything you're confused by or isn't working and want to discuss?
- Questions about an Issue:
    - Alyssa's Guide Pages Prototype: Would like feedback on formatting (Indentation).

**Things discussed:**

- Week Stand-Up:
     - Basu: 
             - We are currently working on the research for the Home Page. 
             - We are done with the interview guides, and the PMs will provide a review. 
             - Working on the demographics. 
             - Ready to interview people. 
             - Would need to clarify with Bonnie to see how we schedule interviewees.
             - We found participants who are interested. Will continue to reach out as well. 
     - Kristine: 
             - Working on the home page.
             - Meeting with Bonnie and new PMs.
             - Sihem became the new advisor for the team.
     - Alyssa: 
             - Working on the guide pages prototype.
             - Collaborating with Bonnie in implementing new guide pages into the prototype for testing.
     - Anh: 
             - Will be working on a copy for the Projects Page.
             - Will create a draft to have something tangible.
     - Simone: 
             - Been working with Basu and the Research Team.
             - Moving along with the research.
- Updated process for completing an issue and milestones.
     - Note: The handle for our design team is @ hackforla/website-design 
- Feedback on Guide Pages Indentation (Alyssa).
- Home Page Discussion and Review:
     - Program Areas Section:
              - How do we make the description for the Program Areas more succinct?
              - Need to have more SVGs for the technologies.
     - Impact/About Us Section:
              - What needs to be in the section to make it an About Us Section?
     - Communities of Practice:
              - Find icons for the CoPs.
              - Issue for this has been created. Currently in the New Issue Approval. 
              - Ava will have the dev team help to find the logos.
                        - Note: Since we don't have the mobile design yet, we will collect the largest SVGs.     
     - Wins Page.
- Home Page Figma Design of the [current version](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=9477%3A88938) in the red box.

**Actions to take:**

- [ ] Reach out to more people to interview via Slack General Channel. (4-5 People)
- [ ] Find as many SVGs for the technologies under the Program Areas Section.
- [ ] New issue to find icons for the Community of Practice. #2130

**Comment by kristine-eudey** [comment]
**Meeting:** 8/26/21

**Participants:** Kristine, Basu, Alyssa, Janie, Anh, Simone, Sophia, Saasha, Ava, and Pamela

**Things to discuss:**

- Weekly standup: What progress have you made on issues you're working on in the past week? What are your goals for the next week? Any questions / anything you'd like feedback on?
- New issues in need of designers
- Homepage Redesign - Current version & planning for mobile
- Guide Pages Inquiry: What is the Header? (Alyssa)

**Things discussed:**
- Weekly Standup:
     - Kristine: 
              - Working on the homepage. 
              - Close to having the desktop version done. 
              - Will be working on mobile soon.
     - Basu: 
              - Doing progress with research team. 
              - Reviewed PM team suggestions. 
              - Done with editing according to suggestions. 
              - Will comment on why we take certain suggestions and why we did not take certain suggestions.
              - Will start on the interview process.
     - Alyssa: 
              - Working on redesigning the Guide Pages.
              - Needs feedback about the Header from the Design Team. 
     -  Janie:
              - Applied feedback on Program: Citizen Engagement from Sunday's meeting.
              - Will create a new mockup to share on Sunday.
     -  Anh:
              - Applied feedback on Project's Page from Sunday's meeting.
              - Note: According to the research, our prospective volunteers are more drawn to the projects. 
              -  Discussed the CTA (Join Us Page). (Our team voted for the Join Us Button.)
     -  Simone:     
               - Been working with the Research team.
     -  Sophia:     
               - Finished Share your Wins form. Ready for review.
               - Issue is in UAT. #2021 
     -  Saasha:     
               - Discussed the Mega Footer. 
               - Feedback: Did a number of versions. Go with the sentence case.
- Discussed who is available to take on new issues.
- Homepage Redesign - Current version & planning for mobile
     -  Updates:     
               - New menu bar with the gray box.
               - Added a section: Explore our Technologies.
               - Rewording on the Wins Section.
     -  Home Page Mobile:     
               - Anyone interested in building ideas starting next week? Anh, Alyssa, Janie
                
**Actions to take:**
     -  Home Page Mobile: Anh, Alyssa, Janie are interested in ideating for Home Page Mobile.

**Comment by kristine-eudey** [comment]
**Meeting: 9/2/2021**

**Participants:** Ashley, Ava, Kristine, Alyssa, Simone

**Things to discuss:**
- Weekly standup - What progress have you made on issues you're working on in the past week? What are your goals for the coming week?
- We need to bring more designers onto the team! Know anyone? 
- Figma organization & best practices / aligning the team on the convention for organizing our designs in Figma 
   - Everything needs to be in a linear path. Whole path should readable from top to bottom, left to right.
   - Each new fidelity level goes down a line
   - Most updated version on the right. 
   - Current design should always be bottom right, with a pink box around it. Did you make a change that was signed off on and updates the current design? Move the pink box to it. Let's minimize using differently colored boxes. 
  
- CoP icons on the [homepage](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=14637%3A94968) - feedback?
- Donor/Partner Usability test brainstorm
- Pressing Issues:
  - #2190 
  - #2213 

**Things discussed:**
- Weekly Stand-Up
   - Kristine:
      - Working on the Home Page.
      - Creating issues and participating in the team meetings.
      - Cleaning up the Figma.
      - Putting a group together to work on the Home Page.
   - Alyssa:
      - Finalized redesign of the Guide pages.
      - Working on developing the pages.
      - Working on documentation for research on cloning our Hack for LA website with Netlify.
    - Ashley
       - Will work on the audit of the mobile with the team.
       - Will work with Kristine in cleaning up the Figma to show the current version.
- Figma organization & best practices / aligning the team on the convention for organizing our designs in Figma 
   - Everything needs to be in a linear path. Whole path should readable from top to bottom, left to right.
   - Decided to uses boxes to show a linear path. Just need to have colors to denote what each boxes mean.
   - Pink Boxes are current one.
   - Make sure that everything that's current reflects what we have in the Current Page.
![image](https://user-images.githubusercontent.com/38295612/131932162-177121e1-610b-4d8b-9aae-faf7969fa92e.png)
- Homepage
   - Would like to have UX Designers to be part of the usability testing.
   - Discussed Program Area Section.
   - Discussed the section: "Explore Our Projects by Technologies." 
   - Discussed testimonials, CoP. 
      - Looking into other icons for CoP.
   - Discussed wins sections. 
      - Wins would be randomized.
      - What is the "see more" going to do when someone clicks on it? 
         - It would lead to the Wins Page. Another idea would be have the wins card take the users to the person's LinkedIn.
         - The "see more" in the Wins card will be reflective with the 
   - Feedback from the team:
      - Are we planning to use a different font for this section? 
         - It's still Roboto, just bigger.
         - _Learn More_ and _Join Us_ buttons may be confusing because it looks alike, even though it will lead to different pages.
             - We could swap the sections around so that the call to actions (to Join Us) would stand out.
         - Create a style sheet that would introduce [new font sizes](https://user-images.githubusercontent.com/38295612/131932720-33d48319-44b1-44cc-b528-f9db42561aeb.png).
      -  Are we on the final stages of Home Page (and will there be more changes in the future)?
      - Our UX/UI team are still working on the home pages so when we have the final product, then there will be no further changes.
- Donor/Partner Usability test brainstorm
   - Reviewed the Homepage Redesign Research Plan. [Figma Link](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=9477%3A88938)
   - Structure on the Usability:
       - Test based on sections; encourage the interviewees to share any of their thoughts. 
       - Ask them about terminology.
       - Next Meeting: TBD  
 
**Actions to take:**
- [ ] Brainstorm the Structure of the Usability Testing: Schedule to meet next week. (Labor Day on Monday.)
- [ ] Design Mobile Version for Homepage: Schedule to meet.
- [ ] Clean up Figma.

**Comment by kristine-eudey** [comment]
**Meeting Date:** 9/9/21

**Participants:** Kristine, Anh, Janie, Alyssa, Kristin, Ashley, Simone, Basu

**Things to discuss:**
- Weekly standup - What progress have you made on issues you're working on in the past week? What are your goals for the coming week?
- We need to bring more designers onto the team! Know anyone?
- Figma organization (quick recap for those who missed last meeting)
- Reminder to keep issues you're assigned to updated weekly. 
- Usability test planning meeting - Monday 11:30am PST
- Issues to assign:
   - #916 
   - #1841 
   - #2207 
   
**Things discussed:**
- Introductions
- Discussed if anyone know any designers to join our team.
  - Alyssa knows someone who is interested - will update when colleague returns from out of town.
- Weekly standup
  - Kristine: 
       - Met with Bonnie to organize the project board.
       - Home Page is under construction, but everything is going well.
       - There is a section that needs improvement - hoping next week that we will have a final version.
       - Janie and Alyssa will work on the mobile version of the home page.
       - Will work on planning the usability test on the home page for donors and partners. 
   - Basu: 
       - To start the page, should it be clickable on the mockups? In a perfect world, it would great. Realistically, the resources and time may not be feasible.
       - _Usability test planning meeting - Monday 11:30am PST_
           - Our main goal is to create questions that we can use for donors and partners.
           - Focus by sections.
           - To start the page, should it be clickable on the mockups? 
                - In a perfect world, it would great. 
                - Realistically, the resources and time may not be feasible.
       - Are we recruiting anyone else aside from donors and partners?
          - Yes, anyone who is focused on giving would be helpful.
   - Janie:  
       - Updated the mobile version on the mobiles page and moved it to UAT.
       - Grabbed a new issue regarding the 404 page.
   - Alyssa:  
       - Redesigning Guide Pages.
       - Would like to collect feedback from team to see if anything needs changes before meeting with Bonnie tomorrow morning. 
       - Would also like to talk about the files.
   - Kristin:  
       - Looking for a new issue!
    - Anh
       - Working on copy for Project Page - there's a direction change in the process.
       - Will work on the mobilizing on the Home Page and audit.
    - Simone
       - Working with the research team in formulating a plan for the usability testing.
    - Ashley
       - Working on the site audit. Will update us with questions.
- Discussed that we need another UX Researcher.
- Reminded the team to keep issues you're assigned to updated weekly. 
    - Please be sure to add a comment with your progress report after assigning yourself with a new issue. 
- Assigned Issues:
   - #916 
       - How to create a video to show how to make issues on GitHub the _wrong way_ with screen-recording.
   - #1841 
      ~~- Need to check the mobile responsiveness on the Donate Page with your phone.~~
       ~~- Check the functionality issues to see how the development worked on mobile.~~
      - **UPDATE:** We will place this in the backburner because we need clarifications on Sunday.
   - **Kristin** -- #2207  
       - Goal is to take the `...see more` and place it at the bottom of the card.
       - Note that it's created with `auto layout`.
       - The `...` will show that there's overflow text and the modal.
       - Other solutions are welcome!
- Guide Pages Redesign Feedback:
   - Discussed Heading: Oswald
       - Since we have Roboto as the text in our Design System, we need to follow it to ensure it's cohesive and create a unity with rest of the website.
   - Discussed spaces between the paragraph and the headings - 36px.
   - Discussed alternative paragraph font size (16px vs 18px). 

**Actions to take:**

- [ ]  Meeting on Monday 11:30AM PST/2:30 PM EST to do usability testing.
- [ ] Will need to reach out to Ryan Curtis and Kevin Wang from Community of Practice to recruit new Designers and Researchers.

**Comment by kristine-eudey** [comment]
**Meeting:** 9/16/21

**Participants:** Kristine, Basu, Anh, Simone, Janie, Alyssa, Ashley

**Things to discuss:**
- Weekly standup
- Sunday meetings 
- Participant recruitment and interview
- Script to recruit corporates
- New researcher recruitment
- Review Issues
   - Guide Pages (Alyssa)

**Things discussed:**
- Weekly standup
   - Kristine
     - Worked on the Home Page.
     - Updating the hero section.
     - Working on sourcing new photos for the Program Areas.
     - Figuring out how to structure our UX Team to focus on outcomes that the users and business would have.
     - Planning to have a meeting where our team will brainstorm how to achieve these goals. 
   - Basu
     - Posted a call out on GitHub and the UI/UX Slack Channel.
     - Will join the Community of Practice to recruit more people.
      - Would like to have someone who has experience with Miro.
     - Will do a rough draft of the script and have it done by Friday. The interview process will be on Sept 28th and 30th.
      - Meeting with UX Research Team on Monday, 9/20/21 at 11:30 AM PST/1:30 PM CST.
    - Simone
      - Worked with Basu on [Homepage Redesign Research Plan](https://docs.google.com/document/d/1ELtk7H_uubatggRcTO9imHzdheeiPWkDLs3KZepMwh0/edit).
    - Janie
      - Worked on the [mobile home page's wins](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=15647%3A100797).
      - Assigned self with 404 page - will work on it soon.
      - Will work on the Get in Touch.
    - Alyssa
      - Worked on the home page mobile.
      - Worked on Guide Pages Redesign.
- Homepage Redesign Research Plan Feedback
   - Comparative Version: Have a grid that would show two versions?
- Home Page Mobile Discussion
   - Reviewed Wins Page Mock-Ups (Janie)
   - Reviewed Hero Graphic and Program Areas (Anh)
   - Reviewed Technology and Communities of Practice Sections.
- Meeting on Sunday
   - Kristine will not be able to attend. If you have issues that need review, please bring it to the meeting on Sunday.  

**Actions to take:**
- [ ] Continue working on the Home Page Mobile.
- [ ] Recruit more UX Researchers for Usability Testing of the Home Page.
   - Next Meeting is on Monday, 9/20/21 at 11:30 AM PST/1:30 PM CST.

**Comment by abenipa3** [comment]
**Meeting:** 9/23/21

**Participants:** Kristine, Simone, Basu, Janie, Alyssa

**Things to discuss:**
- New Member
- Weekly standup
- UX Research: Home Page
- Review Issues

**Things discussed:**
- Danya will onboard in the General HfLA meeting as a UX Designer.
- UX Research Team
  - Need the designs to use for the testing.
  - Interview schedule in this [spreadsheet].(https://docs.google.com/spreadsheets/d/1dbRq7z42hAyRIU75XBDfsaaVAVz8kVbQ/edit#gid=699027266) 
- Janie's Update
  - 404
    - Will work on it soon.
  - Reviewed Home Page's CTA
    - Any way to document behavior?
      - There's no standardized way of doing this. It's totally fine to use annotations and arrows if needed.
      - Updated the mobile for the home page.
     - Reviewed Home Page's Wins
      - Is it possible to see if we limit the number of characters?
      - Is there a best practice in applying spacings on links in mobile?
- Alyssa's Update
  - Reviewed Home Page
  - Reviewed Guides

**Actions to take:**    
- Discuss in the Sunday Meeting: Home Page's Wins
      - Is it possible to see if we limit the number of characters?

**Comment by kristine-eudey** [comment]
**Meeting:** 9/30/21

**Participants:** Kristine, Basu, Anh, Anya, Simone, Ashley, Kristin, Alyssa

**Things to discuss:**

- #1051 
- #1630

**Things discussed:**
- Introductions. Welcome, Anya 🙌 
- Weekly Standup
   - Basu and Simone:
      - Interviews for Home Page Redesign this past week.
      - Usability Testing: If anyone knows a Donor, please contact them to ask if they're interested in being interviewed.
   - Alyssa:
      - Guide Pages (has updates for the team to review).
      - Home Page (continue working on it).
   - Ashley:
      - Was working on the mobile audit.
      - Will transfer file to Hack for La's File.
   - Anh:
      - Currently working with Dev regarding [HfLA Projects' Images](https://www.hackforla.org/projects/).
         - Each project has images for their hero section, but some of the hero images don't work (not much consistency.  
         - Looking into creating a best practice/template for projects to show what kinds of imagery works for the projects.
      - Will need to gather more clarification on Sunday.
   - Kristin
      - Will assign a new issue.      
- Issue Discussion
  - Discussed issue created for Home Page Mobile #2213 
  - https://github.com/hackforla/website/issues/2304
    - What is our role here? 
       - The wins card will be randomized with 10 different wins from different fields.
          - There are hardly any wins submitted by UX/UI people.
       - This issue is about getting people to submit a win.
       - Basu - Reach out to former UI/UX people (current and/or former) and try to get people to contribute wins (tell them we’re going to feature these wins on the homepages. Ask UI/UX CoP leads to doing an @channel to drum up more wins from the community
          - Please feel free to let us know if you need help. 
    - Guide Pages
    - Projects Page
       - Do we want to read it horizontally or vertically?
          -   Team voted for Version 2.
       - What if we add the divider line?
          -   Still voted for Version 2. 
       - Planning to bring the discussion on Sunday with the Website team for extra feedback.
- Assigning Issues
   - #1841 
      - Going through the Donate Page with your phone and write to Development _what is wrong?_
      - Document under the issue. Communicate in any way necessary.
      - You could also use _Inspect_ to test the page.
      - Assigned to Ashley.
   - #2171  
       - Redesign the menus. Needs to get done ASAP.
       - Assigned to Kristin
   - #2312
    
**Actions to take:**

- [ ] Home Page Mobile - Talk about when we could finalize the home page mobile design during our next Thursday meeting.
- [ ] Projects Page - Bring discussion on which mockup looks best with the Website Team on Sunday.

**Comment by kristine-eudey** [comment]
**Meeting:** 10/7/21

**Participants:** Basu, Anh, Simone, Kristin, Kristine, Alyssa, Adam

**Things to Discuss:**
- Naming the HfLA Design System
- Review homepage mobile design
- Design System Inquiry from Guides Team (Alyssa)
   - Accessibility Issue about using strong vs bold and em vs italics [[Resource]](https://help.siteimprove.com/support/solutions/articles/80000448460-accessibility-bold-vs-strong-and-italic-vs-emphasis)
   -  Has the copy text in the design system pages been accessibility tested?
-  Our work / our team - how are things? Anything we can change/improve? _(let's push this topic to next week's meeting since a lot of people can't make it tonight)_

**Things Discussed:**
- Website Team Update: 
  - Ava is no longer the lead - She transferred to Civic Tech Team. 
  - Adam will be the Team Lead for Development. 
- Stand-Up Session
  - Kristine
    - Created a full mock-up of the home page mobile. 
  - Basu and Simone
    - Research Team is currently working on the analysis part of the Home Page.
    -   Simone will be presenting on Sunday.
  - Anh
    - Working on the header issue.
  - Kristin
    - Working on the mobile menus. 
    - Aim to have it done before Sunday.  
  - Alyssa
    - Been working on the Home Page Mobile. Looking for feedback.
    - Got mockup for Guide Pages finalized.
    - Working on development.  
- Home Page Mobile Review
  - Reviewed the following changes made after creating a full-page mockup:
    - Program Areas
      - Discussed mobile carousels for Program Areas.
        - Helpful [article](https://www.nngroup.com/articles/mobile-carousels/) for our references. 
      - Voted for the carousel in Full Mockup#3.
      - Is it possible to show the first five categories and then have a new batch of new categories?
        - Yes, it's possible (Adam)
      - Are there best practices on what the ideal thing is for carousel menus?
      - Create a new mockup with the arrows.
      - Check out Figma App called Figma Mirror, which will help preview mobile mockups.
    - Communities of Practice
      - Concern: Font and Background Color in Project Management may not pass the Contrast Checker.
        - Update:  It passes the WCAG AA, but not the WCAG AAA [[Contract Checker Screenshot]](https://user-images.githubusercontent.com/38295612/136479538-6524f9c5-bf4a-4afd-bb38-a3ce5ffee1cc.png)
         -  Discussed if the categories under CoP should direct us to specific categories (PM, Data Science, etc)?
      - The team agreed to have the links direct the user to the specific categories.
    - Join Us
      - Discussed each section for mockups - is it too long? (Wins)
      - Can we have variations in the card that will minimize the Wins (reduce the number of characters and icons)?
       - Yes, with CSS and JavaScript!
     - Wins Card - Will follow up with Janie.
       - See what we could do a expand button in the wins? 
     - CTA - Try to add Join Us Button back to hero. 
- Voting on Design System Name via Miro Board:
  - Create three sticky notes and add them next to the choices. [[Link]](https://miro.com/app/board/o9J_l6zD0JA=/)
  
**Actions to take:**
- [ ] Wins Card - Will follow up with Janie. 
- [ ] Program Areas - Create a new mockup with the arrows.

**Comment by kristine-eudey** [comment]
### Meeting: 10/14/21

### Participants:
Simone, Basu, Anh, Kristin
### Things to Discuss:
- **Weekly Stand-Up** 
   - *New process*: we will run the weekly stand-up from the In-Progress board, discussing the status of each issue that's currently assigned to someone on the team. The goal of this is to keep better track of in-progress issues, encourage the practice of providing regular updates on issues, and increase engagement with the Github project board. 
   -  ** **Please be sure to provide an update on all issues you're assigned before each week's meeting** ** 
   - Anh: Creating a master template, the banner is changing, wants to talk to the developers this Sunday
   - Adam - Creating an issue about issue #2311
   - Simone and Basu: Analysis for donor interview and in progress, wrapping up our additional interview this week
   - Kristin: Updating the mobile menu
  
      **Current In-Progress issues:**
         -  #2211 
- Not sure about carousal being not clickable at this time
-  Two columns with name and icon
- The card with the drop shadow and the blue one looks clickable
- Can we make the technology clickable? Yes!
-  Kristin liked the top one 
- Anh liked the top and the second one  
         -   #1630:    
- More breathing room
         -   #2304   
-  Please submit your wins so we can feature them in the randomizer on the new homepage. 

- What's the content and size we are looking for UX alumni asked. 

-  We can ask them to write whatever they want but will provide a guide on what we are expecting and we can edit it later
         - #1841 - @ashleylin1 are you working on this? If so please provide an update with ETA, if not please put it back in the prioritized backlog so someone else can work on it.
         - #2171 
         - 
         - #2328 - If you have any ideas for an image that could better represent the "Civic Tech Infrastructure" program area please add it as a comment on this issue!
         - #2190 - There are new next steps on this issue: analyzing the findings, summarizing takeaways and recommendations, and making a quick presentation to share the findings. @ashleylin1 thanks so much for all your work on this one so far, would you like to continue on with the next steps? If so please feel free to (and comment with an update), otherwise you can unassign yourself and put it into prioritized backlog and someone else can take it. *(high priority)*
         - #2311 
No updates, still working on it
- **Design critique / review:** 
   - New nav bar #1256 (Kristine's issue, postpone to next week)
   - Homepage mobile #2213 
      - Please discuss and share thoughts on the padding of the sections. [The current design](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=16865%3A102493) has 48px at top and bottom of sections and 36px between headings and content within sections. This is a divergence from the design system, but if that guidance isn’t working we can update it (it’s a living document that should be responsive as needs change). Do we think there should be even more breathing room? Ideally people would be able to focus one section at a time before scrolling down to the next.

   - ![Screen Shot 2021-10-14 at 12 48 18 PM](https://user-images.githubusercontent.com/16949503/137361576-8c5592b4-b8ec-45c3-9ecf-df4bdc50ba09.png)

   - Any ideas for improvement on the Wins section?
   - ![Screen Shot 2021-10-14 at 12 52 51 PM](https://user-images.githubusercontent.com/16949503/137362251-c8c9bfbc-b49b-4b64-b092-ddea641f13d2.png)

 
- **Jobs to be done**
   - If you haven't done so, take 5 mins to update your issues. 
   - Submit a Win! We need more UX wins to include in the randomizer on the new homepage, please consider submitting!
   - Vote on Design System name: https://app.usabilityhub.com/do/76851981216b/7fc

**Comment by kristine-eudey** [comment]
### Meeting: 10/21/21

### Participants: Simone, Kristine, Basu, Kristin, Adam, Jackie

### Weekly Stand-Up (via [Project Board - In-Progress](https://github.com/hackforla/website/projects/7#column-7198228)) / Updates:
 - Introduction of the participants
 - Job update of Basu
 - How to transition from HfLA to new jobs smoothly (allowing team members to lead)
 - How to include PMs in our meetings?
 - How do you measure success in the job interview - a common interview question and looking for more quantitative data -can 
   PMs help?
 - Kristine helped Jackie to understand how we use github
 - Research team will work individually and collaboratively for next week's presentation on the persona 
 - Kristine worked on the top Nav on the drop-down menu items
 - Kristine and Basu helped Jackie understand what research and design we are doing to improve the homepage
- If you have not updated your issue this week, please take a minute to do so now~

### Design Critique:
- #1256 
- [Homepage Mobile Version](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=16905%3A102823) - Ready to hand off to devs! Great work everyone!

### Actions to take:
- Final round of voting for Design System name- [Vote here](https://app.usabilityhub.com/do/ed3e56977fd8/b9c7)
- **For UX research:** Write an issue for creating a survey that is sent to people when they make a donation to help us better understand the demographics of our donors.
- **Anyone who isn't currently working on an issue:** self-assign one from the top of Prioritized Backlog (if you're able to work on something this week).

**Comment by kristine-eudey** [comment]
### Meeting: 10/28/21

### Participants: Anya, Simone, Jenna, Kristine, Jacqueline, Alyssa, Anh

### Weekly Stand-Up (via [Project Board - In-Progress](https://github.com/hackforla/website/projects/7#column-7198228)) / Updates:

- If you have not updated your issue this week, please take a minute to do so now~
- Github intro / refresher

### Design Critique / Sharing work in progress:

### Items Discussed:

- Welcome to the team, Anya and Jenna!
- UX Team Introduction 🎉 
- Weekly Stand-Up
  - UX Research - Draft Donor Research Project (Simone) [[Link]](https://docs.google.com/presentation/d/1qJZMH92gPwCt3d34Ba5RFvw4ZnH-NAqej2wQRPn5yXE/edit#slide=id.p)
    - Simone will be presenting the summary of our Donor Research Results
    - Team Feedback:
      - The persona will help communicate with the entire Website how we're making design decisions while providing the needs of users.
      - What are the next steps for more donations?
        - Will need to discuss this with Bonnie on how to move forward with this information.
      - Redesign really reflects how we applied users feedback.
      - Insight: Have different web pages/personas that would target different groups of donors.
      - Persona is a great summary! Maybe show a Miro Board that to show that we spoke with the individuals as well. 
      - Keep it concise, some of the texts are long, so maybe shorten it.
      - Have a Persona that would summarize the slides for individuals.
  - Guides Design System
    - We will need to update the design system that will reflect on the new design. It may be a helpful reference for other pages.
      - Here are a couple of resources: [[Figma]](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=17202%3A107428) [[Space Hierarchy Notes]](https://github.com/hackforla/website/issues/1630#issuecomment-954330264)
      - For questions, please reach out to @alyssabenipayo 
  - Github Project Board Review
    - Will discuss this further the next time we meet. 
- No meeting this Sunday! Have a chill Halloween 🎉 
 
### Actions to take:

- UX Research Presentation
  - Make long text in slides concise.
  - See if we could add our Miro Board of our interviewees to the slides.
  - Refer to Team Feedback for more information.
- Guides Design System
  - Create an issue: Updating Guides Design System

**Comment by kristine-eudey** [comment]
### Meeting: 11/4/21

### Participants: Issac, Anh, Kristine, Jenna, Matthew, Simone, Alyssa, Ida, Jackie, Anya, Kristin

### Weekly Stand-Up (via [Project Board - In-Progress](https://github.com/hackforla/website/projects/7#column-7198228)) / Updates:
- If you have not updated your issue this week, please take a minute to do so now~
- Another new teammate, introductions!
- Kristine announcement

### Design Critique / Sharing work in progress:
- Reviewed Anh's Issue: #2129 
  - There are inconsistencies in the spelling for "site map" ("site map" and "sitemap")
  - Add a copy that would notify people that the sitemap would include active pages and ones that are currently in development.
  - Share in the Sunday meeting.
- Reviewed Kristine's Issues: 
  - #2328  
    - One of our research interviewees suggested that we change the image for the Civic Tech section.
  - #2304 
    - How to encourage our volunteers to share their Wins? Could we establish a more clear offboarding?
    - Ideas: Maybe have an exit interview for members to share what works and what didn't work during their time in HfLA     
-  Alyssa's Issue:
   - Guides Pages currently in development
- UX Research Team
    - Currently working on our next target: Corporate Donors.
      - Bonnie will be collecting corporate donors for our research team to interview.
      - After the interview, will compile data for our personas.
    - Next Research on Monday, 11/8    
    - UX Research Results Presentation will be on Sunday, 11/7 


### Items Discussed:
- Team Introductions 🎉 
- Kristine Announcement
  - Last day at the end of November.
  - Thank you for everything, Kristine 😄 ❤️ 
- Weekly Stand-Up
  - High Priority Issue Available: #1398 
    - Updating the Design System
  - UX Research Updates
    - Discussed usability testing.
    - Want to gather quantitative data from interviewees in regards to the hero section.
    - Anyone interested in joining the UX Research Team Meeting?
 - Discussion: Explore the HfLA Google Drive!
   - There are documentations created by current and previous volunteers in HfLA that may be helpful going forward.
     - Documentations, including the Website Roadmap Powerpoint, are available in the Drive.


### Actions to take:
- Interested in becoming a UX (Design) Lead? 
  - Please reach out to Kristine.
- Reach out to Simone if you are interested in joining the UX Research Team.

**Comment by kristine-eudey** [comment]
### Meeting: 11/11/21

### Participants: 

### To Discuss
- Introducing new Design Lead!
- New team member intro
- Proposed change UX Research meeting time
- Weekly Stand-Up (via [Project Board - In-Progress](https://github.com/hackforla/website/projects/7#column-7198228)) / Updates:
   - If you have not updated your issue this week, please take a minute to do so now~
- Design Critique / Sharing work in progress:
- Team Brainstorm in Miro: Generating "_How Might We_" statements to identify opportunities for improving the user experience on the site

### Items Discussed:
- Research meetings time change. Possibly 5:30 Tue (PST). 
- Anya and Jenna had questions on how to work on their issues.
- Jenna noticed the small thumbnail images within the program-area page are inconsistent with each other. (Issue: Review Alt Text for program-area images and ensure that they adhere to WCAG)
- Not enough time for How Might We team workshop. Will continue next week.

### Actions to take:
- Simone will send out a survey to others to choose a time to meet that works for everyone.
- Make a master list of all the pages that are live but need a specific site address to access.

**Comment by kristine-eudey** [comment]
### Meeting: 11/18/21

### Participants: Anh, Jenna, Isaac, Summer, Kristine, Simone, Matthew , Jacqueline

### To Discuss:

- Weekly Standup
- Sharing work completed in the last week


### Items Discussed:
**Issue updates:**
- Isaac learning to create more issues on Kanban board for the team
- Simone met with research team to set intentions and goals for upcoming weeks:
continue with persona creation for different user groups, create a research roadmap
- Summer and Jacqueline reviewing data to create personas
- Matthew creating issues for devs; added label type on issue when assigning; issue for privacy policy page
- Kristine tying up loose ends on issues; creating how-to's section in google shared folder; assisting Isaac to take over as new design team lead

**Review of items in progress:**
- [https://github.com/hackforla/website/issues/2446](url) Isaac and Jenna comparing live site vs. figma files: if the pages aren't live yet, indicate this with the same color as shown in the site map key. We can also add a link to the site page if it is created but not yet live to help anyone navigate to that page.

- [https://github.com/hackforla/website/issues/2328](url) Kristine working on choosing images to represent civic tech infrastructure; some disagreement with current image so Kristine edited ipad image with icons representing Hack for LA

- Jacqueline asking if we can clarify 'what is civic tech?'; how can we best represent that with the imagery? Not all volunteers may be familiar with this terminology. Jacqueline proposing this as a research idea.

- [https://github.com/hackforla/website/issues/2354](url) Jenna to update alt text issue to indicate that images are decorative/null (alt="")

- Anh assigned a new issue

### Actions to take:
- Figma logins are going to be changing to our personal Figma account logins. Kristine and Isaac will add our personal accounts -> keep an eye out for an email inviting you to join the team Figma page!

### **No design team meeting on 11/25 for Thanksgiving**

**Comment by IsaacDesigns** [comment]
# Meeting: 12/2/21

### Participants: Isaac, Simone, Summer, Jenna, Anya, Ida, Stephen, Shruti

### To Discuss:
- New member, Stephen, introduction!!!
- Would like to know where the research documents are located to help inform future designs.
- Figma spellchecker plugin and Google screenshot extension.
- Weekly standup (sharing work in progress and completed in last week).
- Rather than have only the design lead review issues we are going to start having the whole design team review each others issues.

### Items Discussed:
- Simone will reorganize the Research documents in Hack for LA's Google Drive to make them easier to find and use.
- Isaac will research spellchecker plugins for Figma so that all team members can use them and get them installed.

### Actions to take:

**Comment by IsaacDesigns** [comment]
# Meeting: 12/9/21

### Participants: 

### To Discuss:
- New member, Zoe, introduction!!!
- @ExperimentsInHonesty Current Goals:

    - We want the Hack for LA site to better represent the Hack for LA organization, and communicate the value of Hack for LA to all user groups (Donators, Volunteers, Communities of Practice, etc..). We want to get them excited about the organization, its values, and what it has to offer.
    - One problem is that it takes Bonnie a lot of time to manually onboard new volunteers every week, and takes away time from her to work on the more important issues. Because of this we want to make it easier for volunteers and other users to get onboarded through mainly the Hack for LA site.

- Access to Google Analytics
- [Project Board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+design%22) 
- Issue [2525](https://github.com/hackforla/website/issues/2515) walkthrough

### Items Discussed:
- Introductions (for new team member Zoe)
- Goals of the website (+ its revamp)
- Ida: #2501
- Anya: #2513
    - Showed multiple options; group preferred option with accordion
- Isaac: #2515  
    - Awaiting Bonnie to move to prioritized backlog 
- ['Current Pages'](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A0) page on Figma (denotes what pages are currently live on the website)
- Stephen: #2361
    - Currently auditing the usage of our existing text styles within the Figma file

### Actions to take:
- Members are invited to ask Simone for access to Google Analytics for HfLA's website

**Comment by IsaacDesigns** [comment]
# Meeting: 12/9/21

### Participants: 

### To Discuss:

### Items Discussed:

### Actions to take:

**Comment by IsaacDesigns** [comment]
# Meeting: 12/16/21

### Participants: Anya, Simone, Shruti, Ida, Jackie, Stephen, Summer, Isaac

### To Discuss:
- [Project Board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+design%22)
- Prioritized issues - We need to work on the most important prioritized issues first.
### Items Discussed:
• Stephen working on font size and desirability issues. He found that we don't need to add anything new. Can cut down on the fonts. Stephen has color-coded all the titles. Removed unnecessary titles. Standardized the titles and paragraphs. Remaining consistent in the usage of fonts. Discussed different font sizes with the team and which one we could get rid of.
• Issac: Need to stay consistent in the style guide for people coming in later on.
• Stephen will work on the style guide. Issac will make an issue for this.
• Ida: working on site maps and updating them. All that is left is some of the issues and updated status and minor changes.
• Bonnie and Issac will update the status on Figma.
• Issac is done with creating alt text template for the community of practices page.
• Issac: pick the most topmost issues in the prioritized backlog.
• Simone: UX Research updates. Shruti is working on an audit on the current research we already have. Jacqueline is working on google analytics. Summer is working on doing preference testing for two different hero image options.
• Issac: if anyone needs us to talk to Bonnie about any issue, please bring it up with us.
• Anya: the auditing of mobile navigation, is it completed or is it an open issue? Can we find out? The next issue in the prioritized backlog to create a mock video, can there be more clarification on this? Issac will follow up on this to find out whats the latest update on these two issues.
• Issac will follow up with Bonnie to find out if other people in the team can create issues or not
### Actions to take:
- Make an issue to iterate the style guide and Text Styles within Figma so that we can consistently use them when designing new screens in the future.
- Ask Bonnie about any issue in the Prioritized backlog that are difficult to understand.

**Comment by IsaacDesigns** [comment]
# Meeting: 12/23/21

### Participants: Jenna, Saumil, Isaac

### To Discuss:
- Anyone working on an issue may iterate it to make more sense.
- Open position as Co-Design Lead! I might be starting a freelance project next week that will take away time from Hack4LA.
- New [Hack for LA Short Term Goals and Strategies doc](https://docs.google.com/document/d/1FxwgdnWBbbQ8Pi3tspnukZTWwXtqnRXfNkNhVMEloY0/edit?usp=sharing)
  - Meant to help focus and prioritize the steps we need to take in order to reach our short term goals.
  - It will help onboard new designers and keep other teams (Devs, PM's etc..) informed of UX teams progress.
  - Will help the design team focus on the big picture and effectively deliver work that reaches these goals efficiently
  - Team members will be able to work on issues they are more interested in. more effectively write issues, and take more ownership of them. 
 #### How can we improve this doc?
Have Deadlines?
Add it to the Project Board?
- [Project Board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+design%22)
### Items Discussed:
- Walked through how the New [Hack for LA Short Term Goals and Strategies doc](https://docs.google.com/document/d/1FxwgdnWBbbQ8Pi3tspnukZTWwXtqnRXfNkNhVMEloY0/edit?usp=sharing) works and it's benefits to the design team and organizational needs and goals.
- Regarding issue deadlines: everyone has different amounts of time to work on things so maybe have a goal deadline for each issue in conjunction with a points system for how long each issue should take. 
Example (If 1 point = 2 hours): After the issue writer lists out each action item they will approximate how many points each one will take and be able to add them all up to determine an expected deadline. If an issue will approximately take 6 hours to complete the points will be 3. We can then determine an expected deadline an issue will be completed by, and list both the total points and expected deadline a team member will be able to finish the issue by.

### Task Items:
- [x] Isaac will try to get in touch with the design lead of another HFLA team to learn how to implement this system.
- [x] Isaac will discuss with Bonnie about allowing team members to write their own issues based on the prioritized tasks in the new [Hack for LA Short Term Goals and Strategy](https://docs.google.com/document/d/1FxwgdnWBbbQ8Pi3tspnukZTWwXtqnRXfNkNhVMEloY0/edit?usp=sharing).

**Comment by IsaacDesigns** [comment]
# Meeting: 12/30/21

### Participants: Ida, Simone, Jenna, Isaac

### To Discuss:
- [x] Current goal [Hack for LA Short Term Goals and Strategy](https://docs.google.com/document/d/1FxwgdnWBbbQ8Pi3tspnukZTWwXtqnRXfNkNhVMEloY0/edit?usp=sharing). We need to improve the design system. Bonnie recommends everyone to watch this video about design systems: https://www.youtube.com/watch?v=_pfyLVXTVSQ
- [x] [Project Board](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+design%22)
- [x] [Check issues team members are assigned to](https://github.com/hackforla/website/issues)
- [x] New points system we are implementing into each issue.
### Items Discussed:
- People that do the Alt Text Audit issues, when finished, will review others who have finished a similar issue.
### Task Items:
- None

**Comment by ExperimentsInHonesty** [comment]
Please review at next team meeting 
https://github.com/hackforla/website/issues/2513
https://github.com/hackforla/website/issues/2497

**Comment by IsaacDesigns** [comment]
### **Meeting: 01/06/22**

**Participants:** Isaac, Stephen, Shruti, Tammy, Jenna, Ida, Jackie, Simone

**To Discuss:**
 
- [x] Have someone take notes for this meeting 
- [x] Welcome new member Tammy!!! 
- [x] Starting February we will start having no weekly meetings for the 1st week of each month. This will help us plan for the month ahead and work an anything else we need to. Because of how slow it gets during the winter holidays we will also start taking the last 2 weeks of the year off.
- [x]  This month the new website will be launched and we will then start improving the user flows for each demographic of users. Before we can work on that we need to get the current issues off the board.
- [x]  Project Board
- [x]  Accountability check

Items Discussed:

**HAPPY NEW YEAR!**

Team introductions - welcome Tammy!

**Review of current project progress:**

1. Ida working on alt text issue for 'wins' page
2. Anya creating mobile top nav - reopened by Bonnie to try and have styling match design guide. Anya using styling from website re-design.
3. Jenna - working on locations issue -rechecking design system and alerting Bonnie if square cards not represented; working on wrapping resource titles on project page - adjusting sizing of cards. Will create a few options and have team vote next week.
4. Tammy - learning how to assign herself to a project on the board. Signing up for spellchecking on Figma.
5. Issac working on next steps for mobile site audit - adding to the design system or figuring out what is missing from the design system for the navigation component.

- Review of accountability checks and point system : 0.5 = approx. 3 hours of work
- Will plan to work on more user flows (ie: donors) after all current issues are finished
- Saumil pointing out heading sizing discrepancy and creating an issue for sizing between headings on different pages.
- Design system currently does not show examples of what size text is being used - may be useful to annotate designs for developers

Task Items:

- [x]  Isaac will update the Current Pages in Figma so that we can finish issue Update Site map #2501
- [x]  Isaac will create an issue to revise the Design Systems Typography section so that we don’t have to use only size and boldness styling as the only text variations between titles and headers.
- [ ] Isaac will start working to improve the onboarding experience of new designers and Design Lead.

**Comment by IsaacDesigns** [comment]
### **Meeting: 01/13/22**

**Participants:**Tammy, Jenna, Simone, Summer, Anya, Isaac, Saumil, Jacqueline

**To Discuss:**
 
- [x] Have someone else take notes for this meeting 
- [x] Unless the research team has anything to share with the design team they will no longer be required to attend Thursday's design meetings. 
- [x] [Designer Accountability check](https://github.com/hackforla/website/issues)
- Anya is working on a research plan to redesign the Mobile navigation menu
- Jenna has designed 3 versions of the [Resources section cards](https://github.com/hackforla/website/issues/2606), and will annotate the  designs for Isaac to present to Bonnie in Sundays meeting.
- Tammy has found a plugin to find the misspelled words (issue [2597](https://github.com/hackforla/website/issues/2597)), and has started creating a spreadsheet of misspelled words to hand off to developers. She was having trouble organizing the spreadsheet in a way that developers will easily be able to spot misspellings on live site. 
- Isaac will add the navigation elements (issue [2190](https://github.com/hackforla/website/issues/2190)) to the design system if they are not there and if they are there, then will add context examples.
- [ ] Isaac is working on writing issues that will improve the design system and would like input from others that may have experience creating or using a Figma Design System.
- https://github.com/hackforla/website/issues/2684
- https://github.com/hackforla/website/issues/2685

**Task Items:**
- [ ]

**Comment by IsaacDesigns** [comment]
### **Meeting: 01/20/27**

**Participants:** Isaac, Jenna, Anya

**To Discuss:**
- [x] The team needs a new design lead because Isaac is leaving in a a few weeks.
- [ ] Summer will present the results of the home screen hero image preference tests.
- [x] [Designer Accountability check](https://github.com/hackforla/website/issues) 
-  
**Task Items:**
- [ ] @jenjen26 will check the contrast accessibility of the new image cards on issue #2497 
- [ ] @AnyaStewart has started on issue #2686

**Comment by IsaacDesigns** [comment]
### **Meeting: 02/10/22**

**Participants:** Isaac, Jenna, Ida, Summer, Bonnie
**To Discuss:**
- [x] Last day for Isaac as team lead.
- [x] Summer will present the results of the home screen hero image preference tests.
- [x] [Designer Accountability check](https://github.com/hackforla/website/issues) 
- [x] Bonnie will discuss next steps.
**Task Items:**
- [ ] Design team needs to figure out what times work for PM and Stakeholders to meet and we need to recruit another design lead.
- [ ] Team and Bonnie discussed what home page hero images we can used based on research teams findings. Research will conduct preference tests on recommended hero images and screenshot of zoom meetings after they have been stitched together.

**Comment by phuonguvan** [comment]
### Meeting: 05/04/22
Participants: Phuong, Bonnie, Sijia, Nathan

Task Items:
- [ ] Review Sijia's lo-fi and hi-fi wireframes #3035
- [ ] Review Johnathan's site map progress #2501
- [ ] Review Citizen Engagement Figma #2694
- [ ] Ask questions regarding #2601 
- [ ] Figure out a backup time meeting for PM and Design to meet if the design lead can't come
